phpBB/includes/mcp/mcp_forum.php

```diff
@@ -275,6 +275,11 @@ function mcp_forum_view($id, $mode, $action, $forum_info)
 			'TOPIC_ICON_IMG_HEIGHT'	=> (!empty($icons[$row_ary['icon_id']])) ? $icons[$row_ary['icon_id']]['height'] : '',
 			'UNAPPROVED_IMG'		=> ($topic_unapproved || $posts_unapproved) ? $user->img('icon_topic_unapproved', ($topic_unapproved) ? 'TOPIC_UNAPPROVED' : 'POSTS_UNAPPROVED') : '',
 			'DELETED_IMG'			=> ($topic_deleted) ? $user->img('icon_topic_deleted', 'TOPIC_DELETED') : '',
+			'S_POST_ANNOUNCE'		=> ($row['topic_type'] == POST_ANNOUNCE) ? true : false,
```

Comment:
Could we please just use `$row['topic_type'] == POST_ANNOUNCE` for these kind of checks.
Without the redundant parentheses `()` and `? true : false`.

Reply:
this is a straight copy paste...so probably outside the scope of this PR to change all however minor. But not my call @phpbb/development-team